src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -203,6 +221,54 @@ const ColorPalette PredefinedColorPalettes::kSeratoDJProHotcueColorPalette =
                 },
                 {0, 2, 12, 3, 6, 15, 9, 14});
 
+// All Rekordbox Palette types, share the same color, but their default colors
+// are not in the selection offered to the user. This usecase is not supported
+// by mixxx's ColorPalette. The compromise is slightly altering the default
+// colors used so they're part of the selection. The difference between these
+// colors should be imperceptible for the unknowing user.
+const QList<mixxx::RgbColor> kRekordboxColorsSelection = {
+        kRekordboxHotcueColor1,
+        kRekordboxHotcueColor2,
+        kRekordboxHotcueColor3,
+        kRekordboxHotcueColor4,
+        kRekordboxHotcueColor5,
+        kRekordboxHotcueColor6,
+        kRekordboxHotcueColor7,
+        kRekordboxHotcueColor8,
+        kRekordboxHotcueColor9,
+        kRekordboxHotcueColor10,
+        kRekordboxHotcueColor11,
+        kRekordboxHotcueColor12,
+        kRekordboxHotcueColor13,
+        kRekordboxHotcueColor14,
+        kRekordboxHotcueColor15,
+        kRekordboxHotcueColor16,
+};
+
+const ColorPalette PredefinedColorPalettes::kRekordboxCDJHotcueColorPalette =
+        ColorPalette(
+                QStringLiteral("Rekordbox CDJ Hotcue Colors"),
+                kRekordboxColorsSelection,
+                {9});
```

Comment:
Do we really need this palette if you can just select green as default hotcue color in the preferences?

Reply:
Good question, I kept it for the sake of completeness, but we can get rid of it if we document that in the preferences.

Reply:
Opinions? @ronso0 @daschuer

Reply:
yeah, let's drop it. we don't need to copy that 1:1, and if that green is vital for ex-RekordBox users ; ) they can easily pick it as default color.